### DISCUSSION TITLE
Grafana dashboard

### DISCUSSION THREAD

**Discussion by bilbogh** [openappsec/openappsec#234] [Feature Requests]
This enhancement is about designing and developing a Grafana dashboard for open-appsec. 
One of the main data sources can be events - https://docs.openappsec.io/references/events-logs-schema

**Comment by Premkrishnavp887** [comment]
Hi bilbiogh,

The Grafana security dashboard for OpenAppSec has been completed and is available here:

🔗 https://github.com/Premkrishnavp887/openappsec-dashboard

The dashboard provides visibility into OpenAppSec security events and operational metrics using Grafana, Loki, Promtail, and Prometheus. It includes attack monitoring, severity analysis, source IP analysis, threat insights, and health monitoring panels.

Feedback and suggestions are welcome.
